shopify_python/google_styleguide.py

```diff
@@ -295,10 +298,20 @@ def __lambda_func(self, node):  # type: (astroid.Lambda) -> None
         elif isinstance(node.body, astroid.BinOp):
             if shopify_python.ast.count_tree_size(node.body) == 3 and len(node.args.args) == 2:
                 node = node.body
-                operator = self.BINARY_OPERATORS[node.op]
+                operator = self.BINARY_OPERATORS.get(node.op, None)
```

Comment:
It is in fact covered by https://github.com/Shopify/shopify_python/pull/74/files#diff-91e31ad8c2d48b1c0d10cce924bc746eR293